### PR TITLE
update `wgpu` to `0.13`

### PR DESCRIPTION
**Pull request by Cupnfish**

### CODE REVIEW

File: wgpu/Cargo.toml [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+qr_code = [ "iced_graphics/qr_code" ]
+default_system_font = [ "iced_graphics/font-source" ]
+spirv = [ "wgpu/spirv" ]
+webgl = [ "wgpu/webgl" ]
 
 [dependencies]
-wgpu = "0.12"
-wgpu_glyph = "0.16"
+wgpu = "0.13"
+wgpu_glyph = { version = "0.16", git = "https://github.com/Cupnfish/wgpu_glyph.git", branch = "wgpu-0.13" }
```
Comment (Cupnfish) [inline wgpu/Cargo.toml:32]: Waiting to merge this: https://github.com/hecrj/wgpu_glyph/pull/89

File: wgpu/Cargo.toml [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+ico = [ "image_rs/ico" ]
+bmp = [ "image_rs/bmp" ]
+hdr = [ "image_rs/hdr" ]
+dds = [ "image_rs/dds" ]
+farbfeld = [ "image_rs/farbfeld" ]
+canvas = [ "iced_graphics/canvas" ]
+qr_code = [ "iced_graphics/qr_code" ]
+default_system_font = [ "iced_graphics/font-source" ]
+spirv = [ "wgpu/spirv" ]
+webgl = [ "wgpu/webgl" ]
```
Comment (hecrj) [inline wgpu/Cargo.toml:28]: Can we remove these formatting changes?

File: examples/integration_wgpu/src/main.rs [hunk 3]
```diff
@@ -262,12 +258,8 @@ pub fn main() {
                          );
 
                         // And recall staging buffers
-                        local_pool
-                            .spawner()
-                            .spawn(staging_belt.recall())
-                            .expect("Recall staging buffers");
+                        staging_belt.recall();
 
-                        local_pool.run_until_stalled();
```
Comment (hecrj) [inline examples/integration_wgpu/src/main.rs:270]: No more local pool! That's great 😄

File: wgpu/src/shader/image.wgsl [hunk 4]
```diff
@@ -2,26 +2,26 @@ struct Globals {
     transform: mat4x4<f32>;
 };
```
Comment (PolyMeilex) [inline wgpu/src/shader/image.wgsl:3]: This has to be updated to new syntax

**Review by PolyMeilex — CHANGES_REQUESTED** [review]

**Comment by PolyMeilex** [comment]
Also, `;` after struct defs is no longer needed (rust like syntax), but that's just a purely stylistic choice. I'm just noting that switching to that syntax is now possible.

**Review by hecrj — APPROVED** [review]
I made some small changes here and there, but everything looks great! Thanks for taking care of this so quickly :bow:

Also very glad to see all of the `wgpu` improvements! :tada:

**Review by PolyMeilex — APPROVED** [review]
No more comments from me, looks good to me as well.

**Pull request merged**